packages/commonwealth/server/routing/router.ts

```diff
@@ -214,7 +216,12 @@ function setupRouter(
     updateAddress.bind(this, models)
   );
   router.get('/domain', domain.bind(this, models));
-  router.get('/status', status.bind(this, models));
+  router.get('/status', cacheDecorator.cache(60, defaultUserKeyGenerator), status.bind(this, models));
+  router.get(
```

Comment:
Is this testing code? Shouldn't we ensure that testing code is not live in production?